docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -43,40 +87,42 @@ tasks.named('compileKotlin', org.jetbrains.kotlin.gradle.tasks.KotlinCompilation
 </tab>
 </tabs>
 
-### Target the JVM
+### Target level
 
-JVM compilation tasks are called `compileKotlin` for production code and `compileTestKotlin`
-for test code. The tasks for custom source sets are named according to their `compile<Name>Kotlin` patterns.
-
-The names of the tasks in Android Projects contain [build variant](https://developer.android.com/studio/build/build-variants.html)
-names and follow the `compile<BuildVariant>Kotlin` pattern, for example, `compileDebugKotlin` or `compileReleaseUnitTestKotlin`.
-
-For both the JVM and Android projects, it's possible to define options using the project Kotlin extension DSL:
-
-<tabs group="build-script">
-<tab title="Kotlin" group-key="kotlin">
+You can configure compiler options for the JVM/Android and Kotlin Multiplatform targets
+in the `compilerOptions {}` block inside the `target {}` block:
```

Comment:
`target {}` DSL is valid for JVM/Android, but not valid for Multiplatform - in KMP it should be specific target name, e.g. `jvm { compilerOptions {}}`.